clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/EcsSpec.java

```diff
@@ -0,0 +1,175 @@
+      "aws.defaultRegions[0].name = us-west-2",
+      "aws.defaults.iamRole = BaseIAMRole",
+      "ecs.enabled = true",
+      "ecs.primaryAccount = ecs-account",
+      "ecs.accounts[0].name = ecs-account",
+      "ecs.accounts[0].awsAccount = aws-account"
+    })
+public class EcsSpec {
+
+  private final Logger log = LoggerFactory.getLogger(getClass());
+
+  @Value("${ecs.enabled}")
+  Boolean ecsEnabled;
+
+  @LocalServerPort private int port;
+
+  @Test
+  public void configTest() {
+    log.info("This is the port test! The PORT is {}", port);
+    assertTrue(ecsEnabled);
+  }
+
+  // TODO set up cached values, debug /ops, remove info logs
+
+  @Test
+  public void listCredentialsTest() {
+    // given
+    String url = getTestUrl("/credentials");
+
+    // when
+    Response response = get(url).then().contentType(ContentType.JSON).extract().response();
+    log.info(response.asString()); // returns 2 entries, aws-account & ecs-account
+
+    // then
+    assertNotNull(response); // TODO inspect response contents
+  }
+
+  @Test
+  public void listLoadBalancersTest() {
+    // given
+    String url = getTestUrl("/ecs/loadBalancers");
+
+    // when
+    Response response = get(url).then().contentType(ContentType.JSON).extract().response();
+    log.info("LOAD BALANCERS response:");
+    log.info(response.asString()); // returns empty list
+
+    // then
+    assertNotNull(response); // TODO inspect response contents
+  }
+
+  @Test
+  public void listServerGroupsTest() {
+    // given
+    String url = getTestUrl("/serverGroups");
+
+    // when
+    Response response =
+        given()
+            .param("cloudProvider", "ecs")
+            .param("applications", "ecs")
+            .when()
+            .get(url)
+            .then()
+            .contentType(ContentType.JSON)
+            .extract()
+            .response();
+    log.info("SERVER GROUPS response:");
+    log.info(response.asString()); // returns empty list
+
+    // then
+    assertNotNull(response); // TODO inspect response contents
+  }
+
+  @Test
+  public void createServerGroupOperationTest() throws IOException { // no response
+    // given
+    String url = getTestUrl("/ecs/ops");
+    String requestBody = generateStringFromTestFile("/createServiceOperation.json");
+    log.info("request body:");
+    log.info(requestBody);
+
+    // when
+    Response response =
+        given()
+            .contentType(ContentType.JSON)
+            .body(requestBody)
+            .when()
+            .post(url)
+            .then()
+            .contentType("")
+            .extract()
+            .response();
+
+    log.info("request response:");
+    log.info(response.asString());
+
+    // then
+    assertNotNull(response); // TODO inspect response contents
+  }
+
+  private String generateStringFromTestFile(String path) throws IOException {
+    return new String(
+        Files.readAllBytes(Paths.get("src/integration/resources/testoperations" + path)));
```

Comment:
I'd recommend using:

```java
Paths.get("src/integration/resources/testoperations", path)
```

So you don't have to worry about the path separator, `get` will make sure to build a valid path. You can use `createServiceOperation.json` instead of `/createServiceOperation.json` if you prefer, both should work.

It might help readability if you create a private static final field, e.g. `TEST_OPERATIONS_LOCATION`, with the base path.